_includes/header.html

```diff
@@ -42,17 +42,14 @@
     <div class="flex items-center md:ml-12">
```

Comment:
I don't think this will adjust in a responsive way when viewing on Mobile. Have you tested on Mobile view?